### DISCUSSION TITLE
how to download or just a web app?

### DISCUSSION THREAD

**Discussion by justforuse** [toeverything/AFFiNE#62] [General]
I could not find the link😿

**Comment by alt1o** [comment]
We don't have an online demo site yet, so for now you can clone the code and launch it locally to experience the results. more documents: https://affine.gitbook.io/affine/basic-documentation/contribute-to-affine/quick-start

**Reply by justforuse** [reply]
ok, I will try it

**Comment by alt1o** [comment]
https://github.com/toeverything/AFFiNE/discussions/95

Many thanks to Lee for his brilliant job! He just made Affine available on MacOS desktop! If you want to have a peek at current affine demo now, please check it out at: https://github.com/m1911star/affine-client

**Comment by stoefln** [comment]
Don't you need a server for team collaboration?
I'd really like to understand the architecture of the project.

**Reply by darkskygit** [reply]
our architecture supports offline point-to-point collaboration, and there is a todo in our roadmap for point-to-point collaboration.
in the next one to two weeks, we will first provide a local data storage solution, allowing you to use our web demo version while the data is still stored locally.
additionally, we have rust-based collaboration libraries under active development, they will be merged into this repository when they are relatively stable.
